extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -517,64 +517,94 @@ export class AmpPanZoom extends AMP.BaseElement {
       }
     });
 
-    this.gestures_.onGesture(PinchRecognizer, e => {
-      const {
-        centerClientX,
-        centerClientY,
-        deltaX,
-        deltaY,
-        dir,
-        last,
-      } = e.data;
-
-      this.onPinchZoom_(centerClientX, centerClientY, deltaX, deltaY, dir);
-      if (last) {
-        this.onZoomRelease_();
-      }
-    });
+    this.gestures_.onGesture(PinchRecognizer,
+        this.handlePinch.bind(this));
 
     if (!this.disableDoubleTap_) {
```

Comment:
The reason why we don't register the double tap is that registering a doubletap in Gestures causes a 200 ms delay to tap events. The purpose of disabling it is to avoid the 200ms delay.

Reply:
I see, that makes sense. A comment would be nice if we do want to go back and make this work with bind / amp-script in the future. Seems like if that attribute changes, we would need to register/unregister this one gesture.

Reply:
Done